.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Adding this config file so we can leverage the same options on the scripts & code editors